projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -157,7 +157,7 @@ export type SocialSettingsFields = {
 export type ScheduledShare = {
 	id: number;
 	blog_id: number;
-	connection_id: number;
+	connection_id: string;
```

Comment:
This is a number, not a string. We have messed up types everywhere.

You may need to use `connectionIds.map(Number).map(...)` to deal with this